system_tests/test_vsphere_local_blueprints/__init__.py

```diff
@@ -82,7 +84,7 @@ def check_vm_name_in_runtime_properties(runtime_props, name_prefix, logger):
     check_name_is_correct(name, name_prefix, logger)
 
 
-def check_correct_vm_name(vms, name_prefix, logger):
+def check_correct_vm_name(vms, name_prefix, logger, hyphens=2):
```

Comment:
Yeah, I put it in since the expected number of hyphens needed to be put in a variable somewhere to make it clearer what was being checked, and it seemed likely that someone would want to create a test which would have a different amount of hyphens at some point.

I could pull it into the method itself rather than the signature, but this feels like it's at least as well placed in the signature?